packages/documentation-framework/scripts/webpack/getHtmlWebpackPlugins.js

```diff
@@ -22,7 +22,8 @@ async function getHtmlWebpackPlugin({
       // Don't prerender fullscreen pages (expensive!)
       prerendering: (isProd && !isFullscreen && !url.includes('extensions')) ? await prerender(url, pathPrefix) : null,
       // Don't use GA in dev mode
-      googleAnalyticsID: isProd ? googleAnalyticsID : false,
+      // googleAnalyticsID: isProd ? googleAnalyticsID : false,
+      googleAnalyticsID: googleAnalyticsID,
```

Comment:
Just confirming that you are wanting to leave analytics on in dev, and also that you want to leave line 25 in but commented out.

Reply:
No, thanks for calling this out